pkg/placement/consistent_hash.go

```diff
@@ -28,8 +28,8 @@ const replicationFactor = 10
 // ErrNoHosts is an error for no hosts
 var ErrNoHosts = errors.New("no hosts added")
 
-// PlacementTables is a table holding a map of consistent hashes with a given version
-type PlacementTables struct {
+// Tables is a table holding a map of consistent hashes with a given version
+type Tables struct {
```

Comment:
Nit: I do not feel that `Tables` is not good naming. what about `ConsistentHashTables` ? cc/ @yaron2

Reply:
I like `ConsistentHashTables` naming, let me update now.